packages/charts/src/core/common/scales/domain-calculation/domain-with-ticks.ts

```diff
@@ -0,0 +1,28 @@
+import cloneDeep from "lodash/cloneDeep";
+import { IAxisConfig } from "../../../grid/types";
+import { IAccessors, IChartSeries } from "../../types";
+import { domain } from "../helpers/domain";
+import { DomainCalculator, DomainCalculatorWithTicks, IScale } from "../types";
+
+export const getAutomaticDomainWithTicks = (config: IAxisConfig, axisGenerator: any, domainCalculator: DomainCalculator): DomainCalculator => {
```

Comment:
Should it be this?

```suggestion
export const getAutomaticDomainWithTicks = (config: IAxisConfig, axisGenerator: any, domainCalculator: DomainCalculator): DomainCalculatorWithTicks => {
```

Reply:
I guess it would be a breaking change technically to rename this to `getDomainWithTicksCalculator`